Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -734,6 +733,9 @@ public void ToggleFlowLauncher()
 
         public void Show()
         {
+            string _explorerPath = FileExplorerHelper.GetActiveExplorerPath();
```

Comment:
I think this might be a useful function to expose to all plugins.

Reply:
It seems like this is a utility function? Maybe we can put the helper in Flow.Launcher.Plugin?

Reply:
@taooceros Yes, that sounds useful